joblib/numpy_pickle.py

```diff
@@ -329,44 +338,49 @@ def load_build(self):
         dispatch[pickle.BUILD] = load_build
 
 
-class ZipNumpyUnpickler(NumpyUnpickler):
-    """A subclass of our Unpickler to unpickle on the fly from
-    compressed storage."""
-
-    def __init__(self, filename, file_handle):
-        NumpyUnpickler.__init__(self, filename,
-                                file_handle,
-                                mmap_mode=None)
-
-    def _open_pickle(self, file_handle):
-        return BytesIO(read_zfile(file_handle))
+def _dump(value, f, protocol):
```

Comment:
Doesn't seem that necessary to have a helper function that can be written on one-line.